cloudsync/cs.py

```diff
@@ -260,24 +260,39 @@ def start(self, *, daemon=True, **kwargs):
         self.nmgr.notify(Notification(SourceEnum.SYNC, NotificationType.STARTED, None))
         self.nmgr.start(**kwargs)
 
-
     def stop(self, forever=True, wait=True):
         """
         Stops the cloudsync service.
 
         Args:
             forever: If false is passed, then handles are left open for a future start.  Generally used for tests only.
         """
+        self._stop(forever=forever, wait=wait, join=True)
+
+    @classmethod
+    def stop_all(cls, syncs: List["CloudSync"]):
```

Comment:
I know I'm the one who suggested `classmethod` instead of `staticmethod`, but if you're not going to use the `cls` argument, then you were probably right, and `staticmethod` is the right way to go

Reply:
sure, will do